client/state/activity-log/actions.js

```diff
@@ -158,6 +160,35 @@ export function rewindStatusError( siteId, error ) {
 /**
  * Restore a site to the given timestamp.
  *
+ * @param  {string|number} siteId Site ID
+ * @param  {number}        activityId Activity ID
+ * @return {Object}        action object
+ */
+export function rewindRequestRestore( siteId, activityId ) {
+	return {
+		type: REWIND_RESTORE_REQUEST,
```

Comment:
This is when a user requests a rewind to a particular activity? `REQUEST` is an appropriate term, but we've used it so much for API requests, maybe we should pick something different?

Reply:
I replaced with `_PLEASE` which sounds less like something on the network.